dotnet/src/webdriver/Internal/Base64UrlEncoder.cs

```diff
@@ -23,7 +23,7 @@ namespace OpenQA.Selenium.Internal
     /// <summary>
     /// Encodes and Decodes strings as Base64Url encoding.
     /// </summary>
-    internal static class Base64UrlEncoder
+    public static class Base64UrlEncoder
```

Comment:
Keep it internal, already implemented in `trunk`.

Reply:
I'll rebase on trunk